src/Symfony/Component/Debug/Tests/DebugClassLoaderTest.php

```diff
@@ -170,7 +170,7 @@ public function testDeprecatedSuper($class, $super, $type)
     {
         set_error_handler('var_dump', 0);
         $e = error_reporting(0);
-        trigger_error('', E_USER_DEPRECATED);
+        @trigger_error('', E_USER_DEPRECATED);
```

Comment:
It's a trick to reset the state of error_get_last() a few lines below. Silencing should be reverted here, as it's already done on line 172.